src/object.js

```diff
@@ -37,5 +37,62 @@ _.extend(Marionette.Object.prototype, Backbone.Events, {
   bindEntityEvents: Marionette.proxyBindEntityEvents,
 
   // Proxy `unbindEntityEvents` to enable unbinding view's events from another entity.
-  unbindEntityEvents: Marionette.proxyUnbindEntityEvents
+  unbindEntityEvents: Marionette.proxyUnbindEntityEvents,
+
+  //Proxy Radio message handling to enable declarative interactions with radio channels
+  _radioTypes: {
+    'radioEvents' : {
+      startMethod: 'on',
+      stopMethod: 'off'
+    },
+    'radioCommands' : {
+      startMethod: 'comply',
+      stopMethod: 'stopComplying'
+    },
+    'radioRequests' : {
+      startMethod: 'reply',
+      stopMethod: 'stopReplying'
+    }
+  },
+  //store the radio channels being used
+  _radioChannels: [],
+
+  _proxyRadioHandlers: function() {
+    this._unproxyRadioEvents();
+    _.each(this._radioTypes, function(commands, radioType) {
+      var hash = _.result(this, radioType);
+      if (!hash) {
+        return;
+      }
+      _.each(hash, function(handler, radioMessage) {
+        if (!_.isFunction(handler)) {
+          handler = this[handler];
+        }
+        if (!handler) {
+          return;
+        }
+        var messageComponents = radioMessage.split(' '),
+          channel = messageComponents[0],
+          messageName = messageComponents[1];
+        this._proxyRadioHandler(channel, radioType, messageName, handler);
+
+      }, this);
+  }, this);
+},
+
+  _proxyRadioHandler: function(channel, radioType, messageName, handler) {
+    var method = this._radioTypes[radioType].startMethod;
+    Backbone.Radio[method](channel, messageName, handler, this);
+  },
+
+  _unproxyRadioEvents: function() {
+    _.each(this._radioChannels, function(channel) {
+      _.each(this._radioTypes,function(commands) {
+        Backbone.Radio[commands.stopMethod](channel,this);
```

Comment:
Yep.  I think I actually need to add more arguments here, Backbone events isn't smart enough to figure out that this is supposed to be the context if you only pass one argument.  So this is broken.  But thats the general idea of what I'm trying to achieve.  A small fix will prevent over-unsubscribing on everything but manually subscribed methods on the same object.   Like @jfairbank said, if we want to be airtight we will need to keep track of the callbacks or get the updated radio functions.  It's hardly a disaster as is though.